SmartDeviceLink/SDLFileManager.m

```diff
@@ -179,8 +179,14 @@ - (void)didEnterStateFetchingInitialList {
             BLOCK_RETURN;
         }
 
-        // If there was an error, we'll pass the error to the startup handler and cancel out
+        // If there was an error, we'll pass the error to the startup handler and cancel out other then in case of DISALLOWED
         if (error != nil) {
+            // In the case we are DISALLOWED we still want to transition to a ready state
```

Comment:
```suggestion
            // HAX: In the case we are DISALLOWED we still want to transition to a ready state. Some head units return DISALLOWED for this RPC but otherwise work.
```